ui/webui/resources/br_elements/br_toolbar/br_toolbar.html

```diff
@@ -28,7 +28,8 @@
     display: flex;
     justify-content: center;
     margin: 0;
-    padding: 6px 0;
+    padding-right:1em;
```

Comment:
It is good to change padding from fixed pixel values to relative values. This change makes the layout more responsive.